interfaces/automated-testing/geo-awareness/geo-awareness.yaml

```diff
@@ -147,57 +147,100 @@ components:
       required:
         - uomDimensions
         - verticalReferenceType
-        - verticalReference
-        - longitude
-        - latitude
       properties:
         uomDimensions:
           $ref: '#/components/schemas/UomDimensions'
         verticalReferenceType:
           $ref: '#/components/schemas/VerticalReferenceType'
         verticalReference:
```

Comment:
Is the field name `verticalReference` used in ED-269?  If not, this should probably be something like `heightAboveVerticalReference` rather than simply `verticalReference` (since the reference datum is actually named in `verticalReferenceType` rather than this field)

Reply:
No indeed you are right, it uses the following fields:
- lowerLimit
- lowerVerticalReference
- upperLimit
- upperVerticalReference

`verticalReference` with type `VerticalReferenceType`
and
`heighAboveVerticalReference` with type number would be correct.

Reply:
I've now updated the `verticalReference` field name.